### PR TITLE
docs: add OKRs for WGs

### PR DESCRIPTION
**Pull request by sofianguy**
Adding OKRs pulled from [post-summit](http://bit.ly/ems-nyc-output) outputs

### CODE REVIEW

File: wg-security/README.md [hunk 1]
```diff
@@ -17,6 +17,20 @@ incidents, and oversees rollout of fixes.
 | <img src="https://github.com/ppontes.png" width=100 alt="@ppontes">  | Pedro Pontes [@ppontes](https://github.com/ppontes) | Member | CET (Prague) |
 | <img src="https://github.com/StevenEBarbaro.png" width=100 alt="@StevenEBarbaro">  | Steve Barbaro [@StevenEBarbaro](https://github.com/StevenEBarbaro) | Member | ? |
 
+## Current Objective and Key Results
+**Objective:**
+
+Electron is used/trusted by organizations iwth enterprise and corporate-high-security environments.
```
Comment (malept) [inline wg-security/README.md]:
```suggestion
Electron is used/trusted by organizations with enterprise and corporate-high-security environments.
```

File: wg-security/README.md [hunk 2]
```diff
@@ -17,6 +17,20 @@ incidents, and oversees rollout of fixes.
...
+
+**Key Results:**
+1. Increase adoption of electron security best-practices & tooling in AFP and **partner applications**
```
Comment (malept) [inline wg-security/README.md]:
```suggestion
1. Increase adoption of Electron security best-practices & tooling in AFP and **partner applications**
```

File: wg-security/README.md [hunk 3]
```diff
@@ -17,6 +17,20 @@ incidents, and oversees rollout of fixes.
...
+2. Increase engagement of website security documentation (i.e. MOAR pageviews)
+3. Increase **measurable security** for self-identified enterprise apps.
+
+* _Partner Applications_: an app reporting feedback to Electron but outside the AFP
+* _AFP_: App feedback program
```
Comment (malept) [inline wg-security/README.md]:
```suggestion
* _AFP_: App Feedback Program
```

File: wg-docs-tools/README.md [hunk 4]
```diff
@@ -17,6 +17,16 @@ Oversees externally-focused tooling (e.g. Fiddle, Forge), Electron documentation
 | <img src="https://github.com/nitsakh.png" width=100 alt="@nitsakh">  | Nitish Sakhawalkar [@nitsakh](https://github.com/nitsakh) | Member | PT (San Francisco) |
 | <img src="https://github.com/shiftkey.png" width=100 alt="@shiftkey">  | Brendan Forster [@shiftkey](https://github.com/shiftkey) | Member | AT (Canada) |
 
+## Current Objective and Key Results
+**Objective:**
+
+As a newcomer or experienced app developer, you can use Electron’s tooling and documentation to build your app well.
+
+**Key Results:**
+* Increase adoption of electron-provided tooling.
```
Comment (malept) [inline wg-docs-tools/README.md]:
```suggestion
* Increase adoption of Electron-provided tooling.
```

File: wg-docs-tools/README.md [hunk 5]
```diff
@@ -17,6 +17,16 @@ Oversees externally-focused tooling (e.g. Fiddle, Forge), Electron documentation
...
 
+## Current Objective and Key Results
+**Objective:**
+
+As a newcomer or experienced app developer, you can use Electron’s tooling and documentation to build your app well.
+
+**Key Results:**
+* Increase adoption of Electron-provided tooling.
+* Decrease cases of documented poor-practices - normalized against the randomness and size of the samples
+* Increase views on /docs
```
Comment (codebytere) [inline wg-docs-tools/README.md]: Let's say `https://electronjs.org/docs` for completeness here :)

File: wg-docs-tools/README.md [hunk 6]
```diff
@@ -17,6 +17,16 @@ Oversees externally-focused tooling (e.g. Fiddle, Forge), Electron documentation
...
 | <img src="https://github.com/shiftkey.png" width=100 alt="@shiftkey">  | Brendan Forster [@shiftkey](https://github.com/shiftkey) | Member | AT (Canada) |
 
+## Current Objective and Key Results
+**Objective:**
+
+As a newcomer or experienced app developer, you can use Electron’s tooling and documentation to build your app well.
+
+**Key Results:**
+* Increase adoption of Electron-provided tooling.
+* Decrease cases of documented poor-practices - normalized against the randomness and size of the samples
```
Comment (codebytere) [inline wg-docs-tools/README.md]: I'm not quite clear on what this means?

File: wg-outreach/README.md [hunk 7]
```diff
@@ -15,6 +15,16 @@ Grows the Electron community
 | <img src="https://github.com/erickzhao.png" width=100 alt="@erickzhao">  | Erick Zhao [@erickzhao](https://github.com/erickzhao) | Member | ET (Montreal) |
 | <img src="https://github.com/VerteDinde.png" width=100 alt="@VerteDinde">  | Keeley Hammond [@VerteDinde](https://github.com/VerteDinde) | Member | PT (Portland) |
 
+## Current Objective and Key Results
+**Objective:**
+
+Electron is easy for new people to approach and make meaningful contributions.
+
+**Key Results:**
```
Comment (codebytere) [inline wg-outreach/README.md]: For consistency with other OKRs as well as a bit more elaboration:
```
* Reduce time needed to set up developer environment
* Increase breadth of contributors across organizations
* Increase the number of regular contributors (rolling 90-day active)
```

File: wg-docs-tools/README.md (hunk 6 above)
Comment (ckerr) [reply wg-docs-tools/README.md]: Could be rephrased. The intent was to keep a small sample size of respondents from causing the KR to pass/fail due to statistical noise.

Part of the discussion for this KR was that @felixrieseberg goes around and talks to different Electron app devs and asks them about their experience. That's a good feedback channel, but if we define success based on that sample alone, the KR could pass/fail just based on the luck of the draw of who Felix talked to that quarter.

File: wg-outreach/README.md (hunk 7 above)
Comment (ckerr) [reply wg-outreach/README.md]: :+1: 

The addition of "across organizations" here isn't in the original OKR notes but IMO it's consistent with the governance session where there was concern that any one company could have de facto control over the project

File: wg-docs-tools/README.md (hunk 6 above)
Comment (sofianguy) [reply wg-docs-tools/README.md]: @electron/wg-docs-tools 
I was not at the post-summit meeting. Can someone from this WG clarify this?

Comment (codebytere) [reply wg-docs-tools/README.md]: ah, gotcha! i'd love to make perhaps elaborate a bit more to make this a bit clearer  :)

Comment (codebytere) [reply wg-docs-tools/README.md]: ah, gotcha! i'd love to make perhaps elaborate a bit more to make this a bit clearer  :)

Comment (ckerr) [inline wg-docs-tools/README.md]:
```suggestion
* Decrease percentage of documented poor-practices
```

@codebytere does this resolve your question? I think this is a much clearer way of saying most of what was discussed above.

**Pull request merged**